api/types/sortby.go

```diff
@@ -0,0 +1,22 @@
+package types
```

Comment:
Is `types` the best package for this? This is a pretty low-level commonly imported package. Maybe there's a better place for it?

Reply:
This was first best guess at where to put it. I'm open to suggestions, not quite sure which context suites it best.